src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -62,6 +62,13 @@
 
 redis_kwargs = {'unix_socket_path': '/var/run/redis/redis.sock'}
 
+
+def get_neigh_info(neigh_key):
+    neigh_key = neigh_key[neigh_key.find(':')+1:len(neigh_key)]
```

Comment:
>':' [](start = 41, length = 3)

Instead of hard-coded, could you get it by `get_separator()`. But only call this function minimal times.

Reply:
Fixed this, to only split neigh_key information of IPv4 neigh address.